### DISCUSSION TITLE
How to change a value in odoo.conf and apply it, without building the docker image

### DISCUSSION THREAD

**Discussion by piqoni** [Tecnativa/doodba#480] [Doodba - The image]
Hi there, 
Wanted to change **db_maxconn** number without building the image because the situation requires that we do it multiple times and quickly. Is that possible with doodba? 

What I tried: 
Changed the value in **/opt/odoo/auto/odoo.conf**  **within** the container and then restarted the docker container
`docker-compose restart odoo`
but the change in /opt/odoo/auto/odoo.conf was reverted back when I did the container restart. 

Is there any way to apply changes directly, only by restarting Odoo (maybe within the container or something)?

Thank you

**Comment by flachica** [comment]
You have a configuration [option](https://github.com/Tecnativa/doodba#optodoocustomconfd) to set all environment before you lose your work
